db/migrate/20211109210319_create_bands.rb

```diff
@@ -0,0 +1,12 @@
+class CreateBands < ActiveRecord::Migration[6.1]
+  def change
+    create_table :bands do |t|
```

Comment:
Sorry, one last comment - could we also add the model for bands so that each group has that as well and is not duplicated?